### DISCUSSION TITLE
install ejbca with docker compose

### DISCUSSION THREAD

**Discussion by trantuandatvietnam** [Keyfactor/ejbca-ce#664] [Q&A]
Hello everyone, I am following this tutorial with docker compose to use ejbca, however when running the command `docker compose up -d` no link is displayed
![image](https://github.com/user-attachments/assets/aeafb80d-20e8-4155-94d4-31376738bd60)
I try to follow the instructions but when entering the url into the browser, it returns "Server Not Found"
![image](https://github.com/user-attachments/assets/10a3c6bf-695c-438f-a806-7bde909f8a46)

**Comment by rriemann** [comment]
Can you post your entire docker compose file?

Your screenshot suggests that you attempt to access the service on the standard SSL port 443 when oftentimes EJBCA is using higher port numbers such as 8081/8082/8443.
